en/developer_guide/multiple_device.rst

```diff
@@ -0,0 +1,89 @@
+Multiple device
+===============
+
+Prerequisite
+------------
+
+To enable the multi-device rendering, we assume that the server in front of Php (Varnish, Apache2, Nginx, ...)
+ implements the Wurfl librairy and is able to add a request header with the device type.
+
+The device name will be used in the configuration and in the template name following the rule :
+
+* main template : template.html.twig
+* phone template : template.phone.html.twig
+
+Configuration
+-------------
+
+To activate the adaptative response, you have to define all the devices name in the configuration :
+
+.. code-block:: yaml
+
+    #app\config\config.yml
+    open_orchestra_front:
+        devices:
+            web: ~
+            tablet:
+                parent: web
+            phone:
+                parent: web
+            android:
+                parent: phone
+
+If we take a closer look at this configuration :
+
+* The web device is the root one
+* If there is no android template, it will look for the phone one
+* If there is no phone template, it will look for the main one
+* If there is no tablet template, it will look for the main one
+
+Template engine
+---------------
+
+In order to swith automatically between all the template, we had to add some functionality in the template engine.
+
+If you want to use another template engine, you need to implement these logic. We provide you with all the modification
+we made in twig as an exemple.
+
+Twig exemple
+------------
+
+First we have extended both TwigEngine and TimedTwigEngine in the front bundle :
+
+.. code-block:: php
+
+    FrontBundle\Twig\OrchestraTimedTwigEngine
+    FrontBundle\Twig\OrchestraTwigEngine
+
+Use the ``FrontBundle\Twig\Renderable`` trait which override the ``render()`` method by adding the device name
+in the template name
+
+.. code-block:: php
+
+    if (strstr($name, 'twig'))
+    {
+        return str_replace('html.twig', $device . '.html.twig', $name);
+    }
+
+Then, you declare the service based on the new class and put the templating alias :
+
+.. code-block:: yaml
+
+    #FrontBundle\Resources\config\twig.yml
+
+    parameters:
+     open_orchestra_front.twig.orchestra_twig_engine.class: OpenOrchestra\FrontBundle\Twig\OrchestraTwigEngine
+
+    services:
+        open_orchestra_front.twig.orchestra_twig_engine:
+            class: %open_orchestra_front.twig.orchestra_twig_engine.class%
+            arguments:
+                - @twig
+                - @templating.name_parser
+                - @templating.locator
+                - @request_stack
+                - %open_orchestra_front.devices%
+
+You can have a look on `this pull request`_ handling this modification
```

Comment:
I don't think a PR should be referred to in the docs, as it basically says "go read the code", and can be out of date lateR.